webrender_traits/src/lib.rs

```diff
@@ -27,10 +25,6 @@ extern crate core_graphics;
 #[cfg(target_os = "windows")]
 extern crate dwrote;
 
-#[cfg(feature = "serde_codegen")]
-include!(concat!(env!("OUT_DIR"), "/types.rs"));
-
-#[cfg(feature = "serde_derive")]
 include!("types.rs");
```

Comment:
@emilio I would rather have a WR contributor do that after this PR lands, because each type in `types.rs` should actually live in one of the existing modules.